public/locales/edu/en/translation.json

```diff
@@ -21,10 +21,10 @@
     "links": {
       "contribute": "https://go.gov.sg/go-opensource",
       "directory": "/directory",
-      "faq": "https://guide.for.edu.sg/faq.html",
-      "privacy": "https://guide.for.edu.sg/privacy.html",
-      "terms": "https://guide.for.edu.sg/termsofuse.html",
-      "contact": "https://form.gov.sg/forms/govtech/5c5295edb3d80b0017ac7c81",
+      "faq": "https://guide.for.edu.sg",
+      "privacy": "https://guide.for.edu.sg/privacy-policy",
+      "terms": "https://guide.for.edu.sg/terms-of-use",
+      "contact": "https://form.gov.sg/5c5295edb3d80b0017ac7c81",
```

Comment:
The current links for the privacy policy and terms of use are already working, so there probably shouldn't be a need to change them. (for both Go and For)

Note that the link to the contact us form for the edu variant should link to [http://go.gov.sg/foredusg-contact](http://go.gov.sg/foredusg-contact), instead of the same link that Go uses.